### PR TITLE
Milestone 0.3.1

### PR DESCRIPTION
**Pull request by GitScrum**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -44,6 +44,17 @@ function postHTMLParser(html, options) {
         }
     }
 
+    function normalizeArributes(attrs) {
+        var result = {};
+        Object.keys(attrs).forEach(function(key) {
+            var obj = {};
+                obj[key] = attrs[key].replace(/&quot;/g, '"');
```
Comment (michael-ciniawsky) [inline index.js]: indent ? :)

File: index.js [hunk 2]
```diff
@@ -44,6 +44,17 @@ function postHTMLParser(html, options) {
...
+            Object.assign(result, obj);
```
Comment (michael-ciniawsky) [inline index.js]: `node =< v4.0.0` needs a polyfill 😛

File: index.js (hunk 1 above)
Comment (GitScrum) [reply index.js]: that would be read more conveniently )

File: index.js (hunk 2 above)
Comment (GitScrum) [reply index.js]: Thanks!

**Pull request merged**